addons/cssresources/src/index.ts

```diff
@@ -67,7 +67,7 @@ export const withCssResources = makeDecorator({
   skipIfNoParametersOrOptions: true,
   allowDeprecatedUsage: false,
 
-  wrapper: (getStory: any, context: any, { options, parameters }: any) => {
+  wrapper: (getStory, context, { options, parameters }) => {
```

Comment:
This seems unrelated to your PR, but good catch anyways 👍

Reply:
Just found it when I was looking for `makeDecorator` usage, so I cleaned it ;)